web/src/nav/VersionContextDropdown.tsx

```diff
@@ -0,0 +1,213 @@
+                                                className="btn btn-outline-primary version-context-dropdown__info-dismiss"
+                                                onClick={onDismissInfo}
+                                            >
+                                                Do not show this again
+                                            </button>
+                                        </div>
+                                    )}
+                                    <ListboxList className="version-context-dropdown__list">
+                                        <ListboxGroupLabel
+                                            disabled={true}
+                                            value="title"
+                                            className="version-context-dropdown__option version-context-dropdown__title"
+                                        >
+                                            <VersionContextInfoRow
+                                                name="Name"
+                                                description="Description"
+                                                isActive={false}
+                                                onDisableValue={disableValue}
+                                            />
+                                        </ListboxGroupLabel>
+
+                                        {props.availableVersionContexts
+                                            ?.filter(versionContext => versionContext.name === props.versionContext)
+                                            .map(versionContext => (
+                                                <ListboxOption
+                                                    key={versionContext.name}
+                                                    value={versionContext.name}
+                                                    label={versionContext.name}
+                                                    className="version-context-dropdown__option"
+                                                >
+                                                    <VersionContextInfoRow
+                                                        name={versionContext.name}
+                                                        description={versionContext.description || ''}
+                                                        isActive={props.versionContext === versionContext.name}
+                                                        onDisableValue={disableValue}
+                                                    />
+                                                </ListboxOption>
+                                            ))}
+                                        {props.availableVersionContexts
+                                            ?.filter(versionContext => versionContext.name !== props.versionContext)
+                                            .sort((a, b) => (a.name > b.name ? 1 : -1))
```

Comment:
Maybe we could adapt this sort to avoid the double `filter()` here? I had trouble parsing the two very similar blocks and seeing that this `sort()` call was the only difference.

```suggestion
                                            // Render the current version context at the top, then other available version
                                            // contexts in alphabetical order.
                                            .sort((a, b) => (b.name === props.versionContext || a.name > b.name ? 1 : -1))
```